test/Activity-test.js

```diff
@@ -380,4 +380,61 @@ describe('Activity Repository', () => {
 
     expect(activityRepository.findRecord(7, 'minutesActive')).to.equal(286);
   });
+
+  it('should show a user\'s steps taken, flights climbed, and minutes active for each day in a given week', () => {
```

Comment:
great job! I like how you created an array and assigned it to a variable, then passed that variable in as what the test expects to see.